code/_helpers/names.dm

```diff
@@ -47,8 +47,8 @@ var/religion_name = null
 	return "Tau Ceti"
 
 /proc/commstation_name()
-	if (commstation_name)
-		return commstation_name

```

Comment:
I left the proc name as is and just changed the var.
Optional: also change the helper proc name to dock_name()